examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -102,8 +48,64 @@ const app = {
     console.log("Received Event: " + id);
     console.log("---------");
     Purchases.setDebugLogsEnabled(true);
-    Purchases.configure("api_key");
-  }
+    Purchases.configure("api_key", function() { 
+      console.log("Configured");
+      initializePurchasesSDK();
```

Comment:
this is the big change here - after configure, you get a callback, and you can continue from there